dev/io.openliberty.restfulWS.mpMetrics.filter/src/io/openliberty/restfulws/mpmetrics/MonitorAppStateListener.java

```diff
@@ -61,19 +39,7 @@ public void applicationStarting(ApplicationInfo appInfo) {
          * within an ear file or not.
          */
         String appName = appInfo.getDeploymentName();
-        RestMetricInfo metricInfo = RestfulWsMonitorFilter.getMetricInfo(appName);
-
-        /*
-         * Determine if the application is packaged within an ear file. This is
-         * 
-         * useful since a key created in the RestfulWsMonitorFilter class will be
-         * prefixed with the earname + warname or just the warname. See
-         * JaxRsMonitorFilter class for more information.
-         *
-         */
-        if (appInfo.getClass().getName().endsWith("EARApplicationInfoImpl")) {
```

Comment:
no longer using a filter in this bundle, therefore no longer need to resolve such information.